packages/eslint-plugin-pf-codemods/src/rules/v6/chipReplaceWithLabel/chip-replace-with-label.ts

```diff
@@ -0,0 +1,202 @@
+import { getFromPackage } from '../../helpers';
+import {
+  ImportDeclaration,
+  ImportSpecifier,
+  JSXAttribute,
+  JSXClosingElement,
+  JSXElement,
+  JSXExpressionContainer,
+  JSXFragment,
+  Literal,
+  Node,
+} from 'estree-jsx';
+import { Rule } from 'eslint';
+
+type JSXAttributeValue =
+  | Literal
+  | JSXExpressionContainer
+  | JSXElement
+  | JSXFragment;
+
+const getImportSpecifier = (node: ImportDeclaration, specifierName: string) => {
+  return node.specifiers.find(
+    (specifier) =>
+      specifier.type === 'ImportSpecifier' &&
+      specifier.imported.name === specifierName
+  ) as ImportSpecifier | undefined;
+};
+
+const renameImportSpecifier = (
+  importToRename: ImportSpecifier | undefined,
+  node: ImportDeclaration,
+  fixer: Rule.RuleFixer,
+  newName: string
+) => {
+  if (!importToRename) {
+    return [];
+  }
+
+  const oldName = importToRename.imported.name;
+  const importSpecifier = getImportSpecifier(
+    node,
+    importToRename.imported.name
+  ) as ImportSpecifier;
+
+  const importedName = importSpecifier.imported.name;
+  const name = importedName.replace(oldName, newName);
+  const localName = importSpecifier.local.name;
+  const aliasText = importedName !== localName ? ` as ${localName}` : '';
+  const rename = `${name}${aliasText}`;
+  return [
+    fixer.replaceText(importSpecifier, newName),
+    fixer.replaceText(node.source, "'@patternfly/react-core'"),
+  ];
+};
+
+const getJSXAttribute = (node: JSXElement, attributeName: string) => {
+  return node.openingElement.attributes.find(
+    (attribute) =>
+      attribute.type === 'JSXAttribute' && attribute.name.name === attributeName
+  ) as JSXAttribute | undefined;
+};
+
+const parseBadgeAttributeValue = (
+  badgeAttributeValue: JSXAttributeValue
+): Node => {
+  if (badgeAttributeValue.type !== 'JSXExpressionContainer') {
+    return badgeAttributeValue;
+  }
+
+  const isValueJSX = ['JSXElement', 'JSXEmptyExpression'].includes(
+    badgeAttributeValue.expression.type
+  );
+
+  return isValueJSX ? badgeAttributeValue.expression : badgeAttributeValue;
+};
+
+const moveBadgeAttributeToBody = (
+  node: JSXElement,
+  fixer: Rule.RuleFixer,
+  context: Rule.RuleContext
+) => {
+  const badgeAttribute = getJSXAttribute(node, 'badge');
+
+  return badgeAttribute
+    ? [
+        fixer.insertTextBefore(
```

Comment:
Can we add a space before the text? So the result of the fix is `This is a chip {identifier}` rather than `This is a chip{identifier}`